packages/styles/scss/patterns/logo-grid/README.md

```diff
@@ -0,0 +1,25 @@
+### SCSS
+
+#### Usage
+
+Import the package css into the top of your main CSS file.
+
+```css
+@import '@carbon/ibmdotcom-styles/scss/patterns/logo-grid/index';
+```
+
+#### Elements
+
+| Class                       | Description                                                                     |
+| --------------------------- | ------------------------------------------------------------------------------- |
+| `.bx--logo-grid__container` | Element containing all logo grid rows                                           |
+| `.bx--logo-grid__row`       | Horizontal divisions across the viewport                                        |
+| `.bx--logo-grid__col`       | Vertical divisions across the viewport and change in width at different screen. |  | `.bx--logo-grid__title` | Title for Logo Grid |
+| `.bx--logo-grid__wrapper`   | Wrapper for Logo placeholders                                                   |
+| `.bx--logo-grid__logo`      | Placeholder for logo grid                                                       |
```

Comment:
@mkothur the table here looks off for some reason?

Reply:
Okay, @jeffchew let me fix.